apps/main/src/llamalend/entities/usd-prices.ts

```diff
@@ -1,9 +1,7 @@
-import type { Chain } from '@curvefi/prices-api'
 import { ContractParams, queryFactory, rootKeys } from '@ui-kit/lib/model'
 import { contractValidationSuite } from '@ui-kit/lib/model/query/contract-validation'
-import type { Address } from '@ui-kit/utils'
 
-const { fetchQuery: fetchCoinPrice } = queryFactory({
+export const { useQuery: useTokenUsdPrice } = queryFactory({
```

Comment:
Wait is this entity still required? Can't we use `useTokenUsdRate` from `curve-ui-kit\src\lib\model\entities\token-usd-rate.ts`? Having two token usd queries is a bit ... odd, prob best if we consolidate it? Afaik that query doesn't need the library to be hydrated and queries coingecko too, and there's a PR for Denis to check Prices API as well

Reply:
I don't even know why we have 2 libraries to begin with...

Reply:
we used to have 3, of which 2 have been merged :see_no_evil: 
they are mostly copy paste